projects/duplicate-char-counter/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Duplicate Character Counting in Every Language
+layout: default
+date: 2020-10-04
+last-modified: 2020-10-04
+featured-image: <name of featured image file in assets folder>
+tags: [duplicate-character-counting]
+authors:
+  - GoodbyeBlues
+---
+
+An overview of the duplicate characters counting problem.
+
+## Goal
+
+Although a one line solution for this problem is feasible, just as huge brute force solutions are, we should avoid both of these approaches. Instead, our goal should be to find a fine line between both, where the code is not bogged down but is largely readable and maintainable.
+
+## Requirements
+
+The code should return the count of all duplicate, case-insensitive, alphanumeric characters, that occur more than once in the given string. 
+
+## Testing
+
+|Description| Input              | Output                                    |
+| :-------| :------------------: | ----------------------------------------: |
+| Basic String   | "Goodbye, Blues!"  | 'o' occurs 2 times and 'e' occurs 2 times |
```

Comment:
This is great! I'm wondering if there's a way we can simplify the output for testing purposes. Maybe something like this:

```
o: 2
e: 2
```

Also, how do we know what order this will appear?

Reply:
> This is awesome! I have a quick question about the output. Also, did you mean to delete the article? It looked like there were two of them, but one of them was fine.

Oh the file in _posts folder was just a copy of index.md (ngl i was a little confused yesterday). Should there be a documentation and an article write-up?



> This is great! I'm wondering if there's a way we can simplify the output for testing purposes. Maybe something like this:
> 
> ```
> o: 2
> e: 2
> ```

Absolutely. The actual output from the program in my pull request is:
> Text input: goodbyeblues
> Characters: o, Occurrences: 2
> Characters: b, Occurrences: 2
> Characters: e, Occurrences: 2
I can change the example table to better reflect the actual output I have in the script also.

> Also, how do we know what order this will appear?

The order is based on which letters appear first. The full flow from the program is:
> Text input: abba
> Characters: a, Occurrences: 2
> Characters: b, Occurrences: 2

Reply:
Cool! You don't have to do a write-up. I didn't look that deeply, but it looked like there was one. 

As for the characters/occurrences thing, that looks great. Can you update the document so that output is exact? We use the table to generate tests for all languages in the repo, so whatever people see here is what they'll be expected to replicate. Also, make sure to document the order of characters somewhere (probably in requirements).

Reply:
Side note: we always use command line arguments in this repo, so you may need to change your solution to accept a string on the command line (`program.py abba`) instead of through standard input (`Text input: `).

Reply:
Gotcha. I'll get on that now.

Reply:
All good! Also updated the python script and submitted the changes